testing/tests/DevExpress.ui.widgets/box.markup.tests.js

```diff
@@ -94,17 +94,12 @@ QUnit.module("layouting", () => {
     });
 
     QUnit.test("box must have a correct flex direction on items rendering (T604581)", function(assert) {
-        new Box($("#box"), {
+        const boxInstance = new Box($("#box"), {
             direction: "col",
             items: [{ baseSize: 100 }],
-            itemTemplate: function() {
-                if(this.option("_layoutStrategy") === "flex") {
-                    assert.equal(this._$element[0].style.flexDirection, "column");
-                } else {
-                    assert.ok(true);
-                }
-            }
-        });
+        }).instance();
+
+        assert.equal($(boxInstance._$element)[0].style.flexDirection, "column");
```

Comment:
The _layoutStrategy is fallback for IE. Is this test checked in IE?

Reply:
I have reverted this test back